src/main/java/Greeting.java

```diff
@@ -0,0 +1,30 @@
+class Greeting extends Event {
+    int status;
+
+    public Greeting() {
+        super(false);
+        this.status = -1;
+    }
+
+    public Greeting(int inPlay) {
```

Comment:
Maybe can use a more sensible name for this :grin:
```suggestion
    inPlay
```